containerimages/container-disk-images.md

```diff
@@ -12,3 +12,102 @@ The following images stored at `dockerhub.com/kubevirt` and can be used in Kubev
 
 - <b>fedora-cloud-container-disk-demo</b>  
     Fedora cloud edition image.
+
+- <b>fedora-sriov-lane-container-disk</b>  
+    Fedora cloud edition image with contains all necessary configuration and drivers for sriov lane tests.    
+    - This image contained the packages:  
+        - kernel-modules (includes sriov drivers)  
+        - qemu-guest-agent  
+    - Configurations:  
+        - Enable and start qemu-guest-agent  
+        - Load kernel modules needed for sriov  
+          mlx4, mlx5, i40e, igb  
+
+## How to create new customized container-disk for tests
+
+Work-in-progress PR to automate this process https://github.com/kubevirt/kubevirtci/pull/428
+
+Customize an image can be done by:
+- Editing the image with `virt-customize`  
+  Adding new package to an image for example:  
+  `virt-customize -a "fedora32.qcow2" --install dpdk`
+
+- Spin-up live VM with the image you want using `virt-install`  
+  Once the VM is up apply the changes you want and when done
+  shut down the VM gracefully with `sudo shutdown -h now`
+
+Next, it is necessary to prepare the image so there will be no unique files or configurations   
+so each VM that will be created with the new image will have unique machine-id, mac-address etc.. 
+We can do that with `virt-sysprep`:
+ ```bash
+ virt-sysprep -a fedora32.qcow --operations machine-id,bash-history,logfiles,tmp-files,net-hostname,net-hwaddr  
+ ```
+￼
+
+Once the image is ready it is necessary to convert it to   
+container image with `kubevirt/container-disk-v1alpha` layer, 
+so KubeVirt VM's can consume it according to  
+https://github.com/kubevirt/kubevirt/blob/master/docs/container-register-disks.md
+
+```bash
+cat > Dockerfile <<EOF
+FROM kubevirt/container-disk-v1alpha
+ADD fedora32.qcow2 /disk/
+
+docker build -t kubevirt/fedora-sriov-testing:latest .
+```
+
+
+## Use image in Kubevirt tests
+
+First we need pull the image from the remote registry (or local registry) by adding `container_pull` rule to `WORKSPACE` file:
+```bash
+container_pull(
+    name = "fedora_sriov_lane",
+    # digest = ""
+    registry = "localhost:5000",
+    repository = "kubevirt/fedora-sriov-testing",
+    tag = "latest",
+)
+```
+Once you verified the image works reach out to kubevirt CI maintainers and ask to upload the new image 
```

Comment:
sorry if it is a dumb question, how we verify that the image works?

Reply:
Simplest check you can do is create Kubevirt VM with the customized container-disk and verify it boots and connects to the network.
Next thing to do is run functests with the new image, by manipulating `tests/containerdisks/containerdisks.go` or a specific test code to use the new image.

Reply:
Thanks for explaining